tests/test_user.py

```diff
@@ -625,6 +625,25 @@ def test_get_licenses(self, m):
 
         self.assertEqual(2, len(licenses))
 
+    # resolve_path()
+    def test_resolve_path(self, m):
+        register_uris({"user": ["resolve_path_null", "resolve_path"]}, m)
+
+        full_path = "Folder_Level_1/Folder_Level_2/Folder_Level_3"
+        folders = self.user.resolve_path(full_path)
+        folder_list = [folder for folder in folders]
+        self.assertEqual(len(folder_list), 4)
+        self.assertIsInstance(folder_list[0], Folder)
+        for folder_name, folder in zip(("my_files/" + full_path).split("/"), folders):
```

Comment:
The `zip` call makes the Iterable in this for-loop pretty tricky to read - could we pull the resulting list out into its own variable?

Reply:
Do you mean pre-defining the `folder_name` list?  So something like:
```
folder_names = "my_files/" + full_path).split("/")
for j in range(len(folder_names):
     folder_name = folder_names[j]
     folder = folders[j]
...
```

Reply:
Yeah, pulling out `folder_names = (my_files/" + full_path).split("/") and then iterating with `for folder_name, folder in zip(folder_names, folders)` seems fine to me.